.github/workflows/deploy.yml

```diff
@@ -184,6 +184,7 @@ jobs:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           preCommands: |
             echo "*** pre commands ***"
+            git config --global --add safe.directory /github/workspace
```

Comment:
Is `/github/workspace` the actual path that we use or just some boilerplate that was in the error message?

Reply:
> Is /github/workspace the actual path that we use or just some boilerplate that was in the error message?

ah... yeah, I'm not sure 😓

we can merge and give it a try?

or I could use `${{ github.workspace }}` instead:
[![Screenshot 2023-01-12 at 11 14 54](https://user-images.githubusercontent.com/61631103/212052622-6395f236-19ac-4957-9b84-92843da8bdb0.png)](https://docs.github.com/en/actions/learn-github-actions/contexts)
?

Reply:
I think @xtuc suggested that it is coming from wrangler?
https://github.com/cloudflare/wrangler-action/blob/master/Dockerfile#L3

Reply:
can't we find another way to getting the git hash?

Reply:
sure, @petebacondarwin suggested that in the chat, let me have a look...